src/routing/approved.rs

```diff
@@ -1918,23 +1937,20 @@ impl Approved {
 
     fn send_direct_message(&self, recipient: &SocketAddr, variant: Variant) -> Result<Command> {
         let message = Message::single_src(&self.node, DstLocation::Direct, variant, None, None)?;
-        Ok(Command::send_message_to_target(
-            recipient,
-            message.to_bytes(),
-        ))
+        Ok(Command::send_message_to_node(recipient, message.to_bytes()))
     }
 
     // TODO: consider changing this so it sends only to a subset of the elders
     // (say 1/3 of the ones closest to our name or so)
-    fn send_message_to_our_elders(&self, msg_bytes: Bytes) -> Command {
+    fn send_message_to_our_elders(&self, msg: Bytes) -> Command {
```

Comment:
I see some place change from `msg` to `msg_bytes`, and some place change from `msg_bytes` to `msg`
any guidline to follow?

Reply:
I guess `msg` is when it is a concrete type, like say `Message` and `msg_bytes` when it's just `Bytes`. But I'm not sure we are being consistent. Not sure it matters too much though, unless we deal with both `Message` and `Bytes` in the same scope.

Reply:
ok. just being curious as seeing both changes within the same PR  :)